data/json/vehicleparts/wheel.json

```diff
@@ -1,4 +1,109 @@
 [
+  {
+    "type": "vehicle_part",
+    "id": "wheel_mount_light",
+    "name": "light wheel mount",
+    "symbol": "-",
+    "color": "light_gray",
+    "location": "axle",
+    "broken_symbol": "X",
+    "broken_color": "light_gray",
+    "damage_modifier": 80,
+    "durability": 120,
+    "description": "A piece of metal with holes suitable for a bike or motorbike wheel.",
+    "item": "wheel_mount_light",
+    "requirements": {
+      "install": { "skills": [ [ "mechanics", 2 ] ], "time": "10 m", "using": [ [ "welding_standard", 3 ] ] },
+      "removal": { "skills": [ [ "mechanics", 2 ] ], "time": "10 m", "qualities": [ { "id": "SAW_M", "level": 2 } ] },
+      "repair": { "skills": [ [ "mechanics", 1 ] ], "time": "10 m", "using": [ [ "welding_standard", 5 ] ] }
+    },
+    "flags": [ "WHEEL_MOUNT_LIGHT", "NEEDS_JACKING" ],
+    "breaks_into": [ { "item": "scrap", "count": [ 0, 1 ] } ],
+    "damage_reduction": { "all": 10 }
+  },
+  {
+    "id": "wheel_mount_light_steerable",
+    "copy-from": "wheel_mount_light",
+    "type": "vehicle_part",
+    "name": "light wheel mount (steerable)",
+    "extend": { "flags": [ "STEERABLE" ] }
+  },
+  {
+    "type": "vehicle_part",
+    "id": "wheel_mount_medium",
+    "name": "hub assembly",
+    "symbol": "-",
+    "color": "light_gray",
+    "location": "axle",
+    "broken_symbol": "X",
+    "broken_color": "light_gray",
+    "damage_modifier": 80,
+    "durability": 120,
+    "description": "Connection where wheels can be connected on. This particular one is fit for normal car wheels.",
+    "item": "wheel_mount_medium",
+    "requirements": {
+      "install": {
+        "skills": [ [ "mechanics", 4 ] ],
+        "time": "45 m",
+        "qualities": [ { "id": "WRENCH", "level": 2 } ],
+        "using": [ [ "welding_standard", 10 ] ]
+      },
+      "removal": {
+        "skills": [ [ "mechanics", 4 ] ],
```

Comment:
I think mechanics 4 is a far too high requirement for a simple sawing off a hub.